### DISCUSSION TITLE
Easy setup installer

### DISCUSSION THREAD

**Discussion by timminator** [VirtualDrivers/Virtual-Display-Driver#172] [Ideas]
Hi there!
Few months ago i created a fork and added an easy setup installer.  It makes it possible to install the Virtual Display Driver by just clicking through a Setup Wizard.
At the authors (@itsmikethetech, @bud3699 ) of this project, are you interested in such an installer for this main project?
You can check my fork out [here](https://github.com/timminator/Virtual-Display-Driver).

**Comment by bud3699** [comment]
Give me a message on discord. My discord is bud3699, lets discuss on there. If you don't have discord, feel free to email me at bud@mikethetech.com
